koans/TrueAndFalseKoans.php

```diff
@@ -7,5 +7,105 @@
 
 class TrueAndFalseKoans extends TestCase
 {
-    // TODO: Add tests for == and ===, != and !== (FALSE, null, 0, "")
+    // Rule: You can only use true or false for the solution
+
+    /**
+     * @testdox 3 and '3' are the same thing, but, the type? (Equal operator)
+     */
+    public function testStringAndIntAreEqual()
+    {
+        $int = 3;
+        $str = '3';
+
+        $this->assertEquals(__, $int==$str);
+    }
+
+    /**
+     * @testdox 3 and '3' are identical, or not? (Identical operator)
+     */
+    public function testStringAndIntAreNotIdentical()
+    {
+        $int = 3;
+        $str = '3';
+
+        $this->assertEquals(true, $int===$str);
+    }
+
+    /**
+     * @testdox It's the same with Not Equal Operator
+     */
+    public function testWeCanUseNotEqual()
+    {
+        $int = 3;
+        $str = '3';
+
+        $this->assertEquals(__, $int!=$str);
```

Comment:
I think that the spacing is not ok. We need a standard:
$int!=$str -> $int != $str